src/components/microlearning/VideoFeedbackCard.jsx

```diff
@@ -0,0 +1,228 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Card, Icon, IconButton, ActionRow, Form, Input, Button,
+} from '@openedx/paragon';
+import { FormattedMessage, useIntl } from '@edx/frontend-platform/i18n';
+import {
+  ThumbUpOutline, ThumbUp, ThumbDownOffAlt, ThumbDownAlt, Close,
+} from '@openedx/paragon/icons';
+import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
+import { VIDEO_FEEDBACK_CARD, VIDEO_FEEDBACK_SUBMITTED_LOCALSTORAGE_KEY } from './constants';
+
+const VideoFeedbackCard = ({
+  videoId, courseRunKey, enterpriseCustomerUuid, videoUsageKey,
+}) => {
+  const intl = useIntl();
+  const [response, setResponse] = useState(null);
+  const [selectedOptions, setSelectedOptions] = useState([]);
+  const [comments, setComments] = useState('');
+  const [showFeedbackCard, setShowFeedbackCard] = useState(true);
+  const [showFeedbackSubmittedCard, setShowFeedbackSubmittedCard] = useState(false);
+  const feedbackLocalStorageKey = VIDEO_FEEDBACK_SUBMITTED_LOCALSTORAGE_KEY(videoId);
+
+  // On component mount, check if feedback has been previously submitted
+  useEffect(() => {
+    const feedbackSubmitted = localStorage.getItem(feedbackLocalStorageKey);
+    if (feedbackSubmitted === 'true') {
+      setShowFeedbackCard(false);
+      setShowFeedbackSubmittedCard(true);
+    }
+  }, [feedbackLocalStorageKey]);
+
+  const handleThumbClick = (feedbackResponse) => {
+    setResponse(feedbackResponse);
+    if (feedbackResponse) {
+      setShowFeedbackCard(false);
+      setShowFeedbackSubmittedCard(true);
+    }
+    sendEnterpriseTrackEvent(
+      enterpriseCustomerUuid,
+      'edx.ui.enterprise.learner_portal.video.feedback.thumb.submitted',
+      {
+        videoId,
+        courseRunKey,
+        videoUsageKey,
```

Comment:
Could we make the key on this `video_usage_key` to match this other PR? https://github.com/openedx/frontend-app-learner-portal-enterprise/pull/1191/files